refactor(app): verify probe presence before pipette calibration and labware position check

ahiuchingau opened the pull request:
<!--
Thanks for taking the time to open a pull request! Please make sure you've read the "Opening Pull Requests" section of our Contributing Guide:

https://github.com/Opentrons/opentrons/blob/edge/CONTRIBUTING.md#opening-pull-requests

To ensure your code is reviewed quickly and thoroughly, please fill out the sections below to the best of your ability!
-->

# Overview
The app now checks for probe presence for pipette before performing pipette calibration and labware position check.
<!--
Use this section to describe your pull-request at a high level. If the PR addresses any open issues, please tag the issues here.
-->

ahiuchingau committed:
use verify tip presence command before begin calibration/labware position check

ahiuchingau committed:
update module calibration as well

ahiuchingau committed:
fix lint errors

ahiuchingau committed:
format

ahiuchingau committed:
unnest nested promises

ahiuchingau committed:
prettier

smb2268 commented on app/src/organisms/PipetteWizardFlows/AttachProbe.tsx:
```diff
@@ -69,65 +73,64 @@ export const AttachProbe = (props: AttachProbeProps): JSX.Element | null => {
+      },
+    ]
     setIsPending(true)
     refetch()
```
I added this refetch in so that we would have the most up to date `attachedInstrument` state to check for the tip. Is that still necessary? Looks like it could be removed.

sfoster1 committed:
Re-add nesting, remove instrumentsquery

The nesting of the promises was on purpose to get the right error screen
to display so reintroduce it, but it works slightly differently now
because we also don't need the instruments query anymore since we were
just using that to get the tip status.

Also add the failure screen to the module calibration

sfoster1 commented:
The nesting definitely ended up being needed but @smb2268 you're right the instruments query was not, so ditched that. Tested on a robot.

smb2268 reviewed (approved):
Changes look good to me! Thanks for updating 🚀
